### PR TITLE
add cask file: "plotdigitizer.rb"

### PR DESCRIPTION
**Pull request by daehyunpy**

### CODE REVIEW

File: Casks/plotdigitizer.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'plotdigitizer' do
+  version '2.6.8'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/plotdigitizer.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by adityadalal924** [comment]
Please [squash your commits](https://davidwalsh.name/squash-commits-git), thanks.

**Comment by vitorgalvao** [comment]
Thank you for squashing. For future reference, when doing requested changes you should comment after, so we get an email to check back.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 8ad8f05b41766f24dac7e58bcfb574fcc0420feb to keep commit history cleaner. Your contribution is still credited to you.

**Comment by hyuna917** [comment]
Thank you for accepting the request :)

**Pull request closed without merging**